dbsnp-importer/src/main/java/uk/ac/ebi/eva/dbsnpimporter/configuration/ImportVariantsJobConfiguration.java

```diff
@@ -48,21 +46,23 @@
     @Qualifier(IMPORT_VARIANTS_STEP_BEAN)
     private Step importVariantsStep;
 
+    @Autowired
+    @Qualifier(IMPORT_SAMPLES_STEP_BEAN)
+    private Step importSamplesStep;
+
 
     @Bean(IMPORT_VARIANTS_JOB_BEAN)
+    @Primary
     @Scope("prototype")
     public Job importBatchJob(JobBuilderFactory jobBuilderFactory) {
         logger.debug("Building '" + IMPORT_VARIANTS_JOB + "'");
 
-        JobBuilder jobBuilder = jobBuilderFactory
-                .get(IMPORT_VARIANTS_JOB)
-                .incrementer(new RunIdIncrementer());
-
-        FlowJobBuilder builder = jobBuilder
-                .flow(importVariantsStep)
-                .end();
+        JobBuilder jobBuilder = jobBuilderFactory.get(IMPORT_VARIANTS_JOB)
+                                                 .incrementer(new RunIdIncrementer());
 
-        return builder.build();
+        return jobBuilder.start(importSamplesStep)
+                         .next(importVariantsStep)
```

Comment:
Because this job is straightforward, with no conditions or parallelism, we can just use [SimpleJobBuilder](https://docs.spring.io/spring-batch/trunk/apidocs/org/springframework/batch/core/job/builder/SimpleJobBuilder.html).